src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -133,7 +133,10 @@ internal Conversions Conversions
         /// Run the nullable walker during the flow analysis passes. True if the project-level nullable
         /// context option is set, or if any file enables nullable or just the nullable warnings.
         /// </summary>
-        private ThreeState _lazyShouldRunNullableWalker;
+        private ThreeState _lazyShouldRunNullableAnalysis;
+
+        // Nullable analysis data for methods, parameter default values, and attributes. Collected during testing only.
+        internal ConcurrentDictionary<object, int>? NullableAnalysisData;
```

Comment:
>int [](start = 46, length = 3)

Please clarify what the object and int are, and make it a doc comment so it appears in intellisense.